components/tag/stories/template.js

```diff
@@ -0,0 +1,77 @@
+import { html } from "lit-html";
+import { classMap } from "lit-html/directives/class-map.js";
+// import { ifDefined } from "lit-html/directives/if-defined.js";
+
+import { Template as Icon } from "@spectrum-css/icon/stories/template.js";
+import { Template as Avatar } from "@spectrum-css/avatar/stories/template.js";
+// import { Template as ClearButton } from "@spectrum-css/clearbutton/stories/template.js";
```

Comment:
Still needed?

Reply:
similar story here - need to use the ClearButton story eventually but wasn't necessary for initial launch